### PR TITLE
Uses edn for saves.  Refactors a few spots in the code.

### PR DESCRIPTION
**Pull request by saralein**

### CODE REVIEW

File: src/tic_tac_toe/announcer.clj [hunk 1]
```diff
@@ -6,11 +6,13 @@
 
 (defn announce-turn
   [player message]
-  (conj message (str (:name player) "'s turn.\n")))
+  (->> (str (:name player) "'s turn.\n")
+       (conj message)))
```
Comment (damonkelley) [inline src/tic_tac_toe/announcer.clj:10]: 🙌

File: src/tic_tac_toe/announcer.clj [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
 
 (defn move-request
   [board]
-  (format human-move (count board)))
+  (->> (count board)
+       (format human-move)))
```
Comment (damonkelley) [inline src/tic_tac_toe/announcer.clj:15]: 🙌

File: src/tic_tac_toe/announcer.clj [hunk 3]
```diff
@@ -20,4 +22,5 @@
 
 (defn gameover
   [winner message]
-  (conj message (winning-prompt winner)))
+  (->> (winning-prompt winner)
+       (conj message)))
```
Comment (damonkelley) [inline src/tic_tac_toe/announcer.clj:26]: 🙌 

**IF** you wanted to take it _one_ step further....

```clojure
(->> winner
     winning-prompt
     (conj message))
```
😸

File: src/tic_tac_toe/read_write/reader.clj [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
-         (#(serializer/deserialize serializer % create-human create-comp))))
+    [this directory filename]
+    (-> (path/generate directory filename ".edn")
+        (slurp)
+        (read-string)))
 
   (save-exists?
     [this directory filename]
-    (.exists (fs/file (str directory "/" filename ".json")))))
+    (-> (path/generate directory filename ".edn")
```
Comment (damonkelley) [inline src/tic_tac_toe/read_write/reader.clj:19]: There is a bit of duplication between `save-exists?` and `load-game`.

File: src/tic_tac_toe/read_write/writer.clj [hunk 5]
```diff
@@ -1,29 +1,24 @@
...
 
-  (serialize-state
-    [this game]
-    (->> (.bundle-state this game)
-        (serializer/serialize serializer)))
-
   (save-game
     [this directory filename game]
-    (let [path (str directory "/" filename ".json")]
+    (let [path (path/generate directory filename ".edn")]
```
Comment (damonkelley) [inline src/tic_tac_toe/read_write/writer.clj:17]: Here too. The part that concerns me is the `".edn"` argument.

**Review by damonkelley — APPROVED** [review]
Nice changes!

**Review by zdavy — APPROVED** [review]

**Pull request merged**